src/jaguar.toit

```diff
@@ -98,23 +101,23 @@ serve arguments:
   if arguments.size >= 1:
     port = int.parse arguments[0]
 
-  image_config := {:}
-  if platform == PLATFORM_FREERTOS:
-    image_config = esp32.image_config or {:}
+  config := {:}
```

Comment:
It would be nice if we could convince `toit.run` to pass the assets correctly, so we don't have to special case it based on the 'arguments' passed to main.

Reply:
Multiple ways to do that.
One would be to just look for an `assets` folder next to the entry-point and to handle this way.

Reply:
It is kind of annoying to have to do it through temporary directories or files. Maybe we can find a different way of passing it. Not entirely sure yet.